felix/policysync/processor_test.go

```diff
@@ -40,6 +40,19 @@ const ProfileName = "testpro"
 const TierName = "testtier"
 const PolicyName = "testpolicy"
 
+func withTimeout(timeout string, fn func()) func() {
```

Comment:
nit, but feels like a time.Duration would be a better UX for this (and should convert easily into a string that Eventually can handle?)

Reply:
yeah, I think it should be `interface{}` which the `Eventually` can swallow.